bgcval2/makeReport.py

```diff
@@ -84,10 +97,10 @@ def addImageToHtml(fn, imagesfold, reportdir, debug=True):
                 fn,
                 newfn,
         ):
-            if debug: print("removing old file", fn)
+            if debug: print("addImageToHtml:\tremoving old file", fn)
             os.remove(newfn)
             shutil.copy2(fn, newfn)
-            if debug: print("cp", fn, newfn)
+            if debug: print("addImageToHtml:\t copy2", fn, newfn)
```

Comment:
time to start fixing this old Python formatting - add a break and tab pls :grin:

Reply:
```suggestion
            if debug: 
                print("addImageToHtml:\t copy2", fn, newfn)
```